src/libos/crates/vdso-time/Cargo.toml

```diff
@@ -7,11 +7,11 @@ edition = "2018"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [features]
-default = ["libc"]
+default = []
 sgx = ["sgx_types", "sgx_tstd", "sgx_libc"]
 
 [dependencies]
-libc = { version = "0.2", optional = true }
+libc = { version = "0.2" }
```

Comment:
It's a problem caused by no_std `async-rt`.
If `libc` is optional, we need specify `libc` by `default` or feature named like `std`.
```
default = ["libc"]
# or
std = ["libc"]
```
In `async-rt` crate, we can enable vdso-time in sgx by `vdso-time/sgx` feature and disable vdso-time `default` feature.  However, it would cause some trouble when `cargo t`, since there are no libc in vdso-time in this case.

Reply:
vdso-time crate cargo.toml:
```
[features]
default = ["std"]
std = ["libc"]
sgx = ["sgx_types", "sgx_tstd", "sgx_libc"]
```

async-rt cargo.toml
```
[features]
default = ["vdso-time/std"]
auto_run = []
sgx = ["sgx_tstd", "flume/sgx", "vdso-time/sgx", "hierarchical_hash_wheel_timer/sgx"]
```